AnnoyancesFilter/sections/popups.txt

```diff
@@ -133,6 +133,7 @@ techmeme.com###nlsb
 kompas.com##.root > div.dark-fixed
 alibabacloud.com##.btn-survey
 multishare.cz##.left-layout-buttons
+odysee.com##.nudge
```

Comment:
I'm not sure that we should hide this element

Reply:
Well, it's still popup :)